experiment/src/cljs/cortex/suite/classify.cljs

```diff
@@ -102,14 +103,26 @@
             current-data @dataset-atom]
         (when-not (= (get dataset-data :update-index)
                      (get current-data :update-index))
-          (reset! dataset-atom dataset-data)))))
+          (reset! dataset-atom dataset-data))))
+  (go (reset! accuracy-atom (<! (model/put "accuracy-data")))))
+
+
+(defn display-accuracy
+  [accuracy-data]
+  [:div.dataset
+   [:table
+    [:thead [:tr [:th "epoch"] [:th "accuracy"]]]
+    (into [:tbody]
+          (for [[i a] (map-indexed vector accuracy-data)]
+            [:tr [:td (inc i)] [:td a]]))]])
 
 
 (defn classify-component
   [& args]
   (let [confusion-atom (atom {})
-        dataset-atom (atom {})]
-    (js/setInterval #(confusion-matrix-update confusion-atom dataset-atom) 1000)
+        dataset-atom (atom {})
+        accuracy-atom (atom {})]
```

Comment:
I was just following what was already there; I believe it's because the larger Clojure community doesn't use `*` the way we do. Instead of appending it to deref-ables, they use it to mean 'helper-of'.